tests/scene/test_camera_3d.h

```diff
@@ -234,7 +234,7 @@ TEST_CASE("[SceneTree][Camera3D] Project/Unproject position") {
 			// Top left.
 			CHECK(test_camera->project_position(Vector2(0, 0), 1.5f).is_equal_approx(Vector3(-5.0f, 2.5f, -1.5f)));
 			// Bottom right.
-			CHECK(test_camera->project_position(Vector2(400, 200), 5.0f).is_equal_approx(Vector3(5.0f, -2.5f, -5.0f)));
+			CHECK(test_camera->project_position(Vector2(400, 200), test_camera->get_far()).is_equal_approx(Vector3(5.0f, -2.5f, -test_camera->get_far())));
```

Comment:
Do the old tests still pass? I think it would probably be worthwhile to keep the old tests in addition to the new tests

Reply:
Yes they do. Ok I'm going to keep them